tutorials/tour/_posts/2017-02-13-mixin-class-composition.md

```diff
@@ -9,28 +9,41 @@ categories: tour
 num: 6
 next-page: anonymous-function-syntax
 previous-page: traits
+prerequisite-knowledge: inheritance, traits, abstract-classes, unified-types
 ---
+Mixin class composition means that a class can reuse the members of one or more mixins (i.e. traits and types).
+
+```tut
+abstract class A {
+  def message: String
+}
+class B extends A {
+  def message = "I'm an instance of class B"
+}
+trait C extends A {
+  def makeMessageLoud = message.toUpperCase()
+}
+class D extends B with C
+
+val d = new D
```

Comment:
I recommend replacing
```
class D extends B with C

val d = new D
```
with
```
val mixedClassInstance = new B with C
```

This simplifies the presentation and points out that 'B with C' is a class.